packages/chat/src/lang/system.ts

```diff
@@ -0,0 +1,38 @@
+import { WebchateLocale } from '../locale/system'
+
+export class WebchatLang {
+  private translations: { [lang: string]: any } = {}
```

Comment:
It would be

```ts
export interface Translation {
  [key: string]: string | undefined
}
```
Translations arent stored in a recursive way. The squash method flattens the object.